library_generation/model/config_change.py

```diff
@@ -143,11 +146,23 @@ def __create_qualified_commit(
         :return: qualified commits.
         """
         libraries = set()
+        file_change_num = len(commit.stats.files.keys())
         for file in commit.stats.files.keys():
-            if file.endswith("BUILD.bazel"):
-                continue
             versioned_proto_path = find_versioned_proto_path(file)
             if versioned_proto_path in proto_paths:
+                # Skip a commit if the only changed file is `BUILD.bazel`
+                # and the change doesn't contain fields used in library
+                # generation, e.g., transport, rest_numeric_enum.
+                if (
+                    file.endswith("BUILD.bazel")
+                    and file_change_num == 1
```

Comment:
What if a commit contains changes to multiple `BUILD.bazel` files? It is a common use case for multi-module library.

Reply:
Good catch, I need to make some changes when a commit contains multiple `BUILD.bazel`.